obspy/core/util/base.py

```diff
@@ -159,6 +156,7 @@ def create_empty_data_chunk(delta, dtype, fill_value=None):
                  mask = ...,
                  ...)
     """
+    import numpy as np
```

Comment:
I don't have a clue about the new failing test. Could you move this numpy import again to the global scope and see if it helps? Numpy is imported at other places in the global scope, so it should not degrade the load time.

Reply:
good idea, indeed, numpy is imported elsewhere anyway